### ISSUE TITLE
Disabling Role Manager seems to break plugin

### ISSUE THREAD

**Issue by crebacz** [caseproof/members#6]
Hello-- 

When unchecking the Role Manager setting and then saving the settings, I had the following error thrown: 

**"Uncaught Error: Call to a member function template() on bool"** 

Specifically, the offending code is in members/admin/class-settings.php within the settings_page method.  

```
public function settings_page() { ?>
 
        <div class="wrap">
            <h1><?php echo esc_html_x( 'Members', 'admin screen', 'members' ); ?></h1>
 
            **<?php $this->get_view( members_get_current_settings_view() )->template(); ?>**
 
        </div><!-- wrap -->
    <?php }
````

Interestingly, the Members side menu disappears from the admin menu as well, with no way to get it back.  Deactivating or reactivating didn't help.  

In exploring the plugin, what I've learned thus far is that the admin menu hooks inside the class-role-new.php class constructor are wrapped in a conditional that calls 'members_role_manager_enabled'.  Obviously, this will return false when the role manager is disabled, and in turn, the Members parent menu page never gets registered or displayed. And with all Members functionality relocated under that parent menu page in 3.0, turning off the Role Manager essentially renders all plugin functionality inaccessible.

**Comment by renventura** [comment]
@crebacz This should be resolved with 8e602e0129ea6346bc5eeaadf45a671d07dc5fa0, if you'd like to check it out and test on your end.

**Comment by crebacz** [comment]
Sorry just saw this.  Looks good. Thank you!
